auth-backends/README.adoc

```diff
@@ -811,4 +811,157 @@ It means that the role `node_all` is not recognized. It is because it is not a r
 
 In this case you should create a custom role (let's say `access_to_node`) with the permission `node_all`
 that you will map to your IdP role `role-oidc-a` and modifying the parameter `mapping.entitlements` in the OIDC config file like so:
-`rudder.auth.oauth2.provider.okta.roles.mapping.entitlements.role-oidc-a=access_to_node`
+`rudder.auth.oauth2.provider.okta.roles.mapping.entitlements.role-oidc-a=access_to_node`.
+
+
+=== OAuth2 tokens : JWT and opaque bearer tokens
+
+OAuth2 tokens are used to grant access to resources, using authentication tokens, serving the purpose of authenticating machines, automated systems, etc.
+There are two types of tokens implemented as authentication backends in Rudder: JWT (JSON Web Tokens) and opaque bearer tokens.
+
+JWT tokens contain user information and permissions in a JSON format. They can be verified using a signature, allowing for local validation without contacting the authorization server.
+
+Opaque bearer tokens do not contain user information. They serve as a reference to session or authorization data stored on the server. To validate these tokens, the resource server must contact the authorization server to obtain user details.
+
+Both are implemented with configuration that is similar to the provisioning of users with OIDC, except that tokens are involved :
+
+* for JWT, Rudder can locally validate the token's signature, ensuring that the token has not been tampered with and that it is issued by a trusted source. This local validation reduces the need for frequent communication with the authorization server, improving performance and reliability.
```

Comment:
We still need to get the signature keys at their remote place to check that they aren't revoked or just actually get them. But yes, the cache is rather long (in hours iirc)